content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -729,7 +729,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   [ServiceTopology](/docs/concepts/services-networking/service-topology/)
   for more details.
 - `SizeMemoryBackedVolumes`: Enables kubelet support to size memory backed volumes.
-  See [volumes](docs/concepts/storage/volumes) for more details.
+  See [volumes](/docs/concepts/storage/volumes) for more details.
```

Comment:
confirmed the old link was going to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/docs/concepts/storage/volumes which was NotFound page.

/lgtm